gym/f110_gym/envs/f110_env.py

```diff
@@ -156,6 +158,13 @@ def __init__(self, render_mode=None, **kwargs):
         except:
             self.integrator = Integrator.RK4
 
+        # observation
```

Comment:
This should probably follow the new config dict we'll be using.
We'll fix after merging the other pr.